docs/src/man/basics.md

```diff
@@ -0,0 +1,1505 @@
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german = copy(german_ref); # It will copy the data frame
+```
+
+Now let's talk about the given code block:
+```julia
+german = CSV.read((joinpath(dirname(pathof(DataFrames)),
+                                 "..", "docs", "src", "assets", "german.csv")),
+                       DataFrame)
+```
+- we are storing `german.csv` file in the `DataFrames.jl` repository to make user's life easier and 
+  avoid having to download it each time;
+- `pathof(DataFrames)` gives us the full path of the file that was used to import `DataFrames`;
+- first we split the directory part from it using `dirname`;
+- then from this directory we need to move to the directory where `german.csv` is stored; we use 
+  `joinpath` as this is a recommended way to compose paths to resources stored on disk in an operating 
+  system independent way  (remember that Widnows and Unix differ as they use either `/` or `\` as path 
+  separator - `joinpath` is a function to make sure we are not running into issues with this);
+- then we read the CSV file; the second argument to `CSV.read` is `DataFrame` to indicate that we want to 
+  read in the file into a `DataFrame` (as `CSV.read` allows for many different target formats of data it 
+  can read-into).
+
+You can see that Julia representation (unlike Python's Pandas) displays the data type of the column,
+In our case it is a `Int64`, and `String`.
+
+To access the columns directly (i.e. without copying) you can use `german.Sex`, `german."Sex"`,
+`german[!, :Sex]` or `german[!, "Sex"]`. The two latter syntaxes are more flexible as they allow
+us passing a variable holding the name of the column, and not only a literal name. Columns name
+can be passed either as a `Symbol` (written as `:Sex`, `:var"Sex"` or `Symbol("Sex")`) or as a string 
+(written as `"Sex"`). Variable interpolation into the string using `$` does not work if you have forms
```

Comment:
I would think it is too much detail here. Do you think a new user will exactly understand what we want to say here?

Reply:
I think every detail is straightforward here.

Reply:
Ah - so we have a different feeling here. For me such things are complex. E.g. it is not immediately obvious for me which of these work (I am listing the ones you have listed as equivalent ways to pass a `Symbol`):
```
german.:Sex
german.:var"Sex"
german.Symbol("Sex")
```
Do you feel that it is useful to teach people about these ways to express a `Symbol` (I am not telling that not - I would like to get a frank opinion from you here as personally I always thought that no one would in practice write e.g. `:vec"x"` if someone can just write `"x"` to get the same effect)

Reply:
Yeah, you are right.